src/project.ts

```diff
@@ -332,6 +364,58 @@ class Project extends ServiceObject {
       }
     );
   }
+
+  getIamPolicy(): Promise<GetIamPolicyResponse>;
+  getIamPolicy(callback: GetIamPolicyCallback): void;
+  /**
+   * @typedef {array} GetIamPolicyResponse
+   * @property {Policy} 0 The new [policy]{@link https://cloud.google.com/resource-manager/reference/rest/Shared.Types/Policy}.
```

Comment:
I don't know that we would say it's a "new" policy. Maybe just "This project's IAM policy."?